### PR TITLE
Move APIs for service pages

### PR DESCRIPTION
**Pull request by npentrel**
Do not review until #3504 has landed

### CODE REVIEW

File: docs/appendix/apis/services/motion.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+title: "Motion Service API"
+linkTitle: "Motion"
+weight: 40
+type: "docs"
+description: "The motion service API allows you to move a machine based on a SLAM map or GPS coordinates or to move a machine's components form one location to another."
```
Comment (npentrel) [inline docs/appendix/apis/services/motion.md:6]:
```suggestion
description: "Give commands to move a machine based on a SLAM map or GPS coordinates or to move a machine's components form one location to another."
```

File: docs/appendix/apis/services/ml.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+---
+title: "ML model service API"
+linkTitle: "ML Model"
+weight: 30
+type: "docs"
+tags: ["data management", "ml", "model training"]
+description: "The  ML model service API allows you to give commands to your ML model service to make inferences based on a provided ML model."
```
Comment (npentrel) [inline docs/appendix/apis/services/ml.md:7]:
```suggestion
description: "Give commands to your ML model service to make inferences based on a provided ML model."
```

File: docs/appendix/apis/services/generic.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
+---
+title: "Generic service API"
+linkTitle: "Generic"
+weight: 60
+type: "docs"
+tags: ["generic", "services"]
+description: "The generic service API allows you to give commands to your generic components for running model-specific commands using DoCommand."
```
Comment (npentrel) [inline docs/appendix/apis/services/generic.md:7]:
```suggestion
description: "Give commands to your generic components for running model-specific commands using DoCommand."
```

File: docs/appendix/apis/services/data.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
+---
+title: "Data Management Service API"
+linkTitle: "Data Management"
+weight: 10
+type: "docs"
+description: "The data management service API allows you to give commands to your data management service to sync data stored on the machine it is deployed on to the cloud."
```
Comment (npentrel) [inline docs/appendix/apis/services/data.md:6]:
```suggestion
description: "Give commands to your data management service to sync data stored on the machine it is deployed on to the cloud."
```

File: docs/appendix/apis/services/base-rc.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
+---
+title: "Base Remote Control service API"
+linkTitle: "Base Remote Control"
+weight: 70
+type: "docs"
+tags: ["base", "services", "rover", "input controller", "remote control"]
+description: "The Base Remote Control service API allows you to get a list of inputs from the controller that are being monitored for that control mode."
```
Comment (npentrel) [inline docs/appendix/apis/services/base-rc.md:7]:
```suggestion
description: "Give commands to get a list of inputs from the controller that are being monitored for that control mode."
```

File: docs/appendix/apis/services/SLAM.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
+---
+title: "SLAM service API"
+linkTitle: "SLAM"
+weight: 60
+type: "docs"
+tags: ["slam", "services"]
+description: "The SLAM service API allows you to get machine's position within a map."
```
Comment (npentrel) [inline docs/appendix/apis/services/SLAM.md:7]:
```suggestion
description: "Give commands to get a machine's position within a map."
```

File: docs/appendix/apis/services/navigation.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
+---
+title: "Navigation service API"
+linkTitle: "Navigation"
+weight: 50
+type: "docs"
+tags: ["navigation", "services", "base", "rover"]
+description: "The navigation service API allows you to define waypoints and move your machine along those waypoints while avoiding obstacles."
```
Comment (npentrel) [inline docs/appendix/apis/services/navigation.md:7]:
```suggestion
description: "Give commands to define waypoints and move your machine along those waypoints while avoiding obstacles."
```

File: docs/appendix/apis/services/vision.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
+---
+title: "Vision service API"
+linkTitle: "Vision"
+weight: 20
+type: "docs"
+tags: ["vision", "computer vision", "CV", "services"]
+description: "The vision service API allows you to get detections, classifications, or point cloud objects, depending on the ML model the vision service is using."
```
Comment (npentrel) [inline docs/appendix/apis/services/vision.md:7]:
```suggestion
description: "Give commands to get detections, classifications, or point cloud objects, depending on the ML model the vision service is using."
```

File: docs/appendix/apis/services/SLAM.md [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+linkTitle: "SLAM"
+weight: 60
+type: "docs"
+tags: ["slam", "services"]
+description: "Give commands to get a machine's position within a map."
+icon: true
+images: ["/services/icons/slam.svg"]
+---
+
+The SLAM service API allows you to get machine's position within a map.
```
Comment (npentrel) [inline docs/appendix/apis/services/SLAM.md:12]:
```suggestion
The SLAM service API allows you to get a machine's position within a map.
```

File: docs/appendix/apis/services/motion.md [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+title: "Motion Service API"
+linkTitle: "Motion"
+weight: 40
+type: "docs"
+description: "Give commands to move a machine based on a SLAM map or GPS coordinates or to move a machine's components form one location to another."
+icon: true
+images: ["/icons/components/arm.svg"]
+---
+
+The motion service API allows you to give commands to your [motion service](/services/motion/) for moving a machine based on a SLAM map or GPS coordinates or for moving a machine's components form one location to another.
```
Comment (sguequierre) [inline docs/appendix/apis/services/motion.md:11]:
```suggestion
The motion service API allows you to give commands to your [motion service](/services/motion/) for moving a machine based on a SLAM map or GPS coordinates or for moving a machine's components from one location to another.
```

File: docs/services/ml/deploy/tflite_cpu.md [hunk 11]
```diff
@@ -158,25 +158,17 @@ You can use one of these architectures or build your own.
...
-
-Configure an `mlmodel detector`.
-
-{{% /manualcard %}}
-{{% manualcard link="/services/vision/mlmodel/"%}}
-
-<h4>Create a classifier with your model</h4>
-
-Configure your `mlmodel classifier`.
+Use your model deployed with the ML model service by adding a vision service that can provide detections or classifications depending on your ML model.
```
Comment (sguequierre) [inline docs/services/ml/deploy/tflite_cpu.md:168]:
```suggestion
Use your model deployed with the ML model service by adding a vision service that can provide detections, classifications, or segmentations depending on your ML model.
```
not sure if accurate-- would this also include point clouds?

File: docs/services/vision/_index.md [hunk 12]
```diff
@@ -92,31 +86,26 @@ Any camera that can return 3D pointclouds can use 3D object segmentation.
...
-## Used with
-
-{{< cards >}}
-{{< relatedcard link="/services/ml/deploy/" alt_title="Machine Learning" >}}
-{{< /cards >}}
-
 ## API
 
-Different vision service models support different methods:
+The vision service supports the following [Vision service API](/appendix/apis/services/vision/) methods:
```
Comment (sguequierre) [inline docs/services/vision/_index.md:99]: (opt) do we have to capitalize vision here?

File: docs/services/vision/obstacles_distance.md [hunk 13]
```diff
@@ -139,3 +139,13 @@ To learn more about how to use segmentation, see the [Go SDK docs](https://pkg.g
 {{% alert title="Tip" color="tip" %}}
 To see more code examples of how to use Viam's vision service, see [our example repo](https://github.com/viamrobotics/vision-service-examples).
```
Comment (sguequierre) [inline docs/services/vision/obstacles_distance.md:140]:
```suggestion
To see more code examples of how to use Viam's Vision service, see [our example repo](https://github.com/viamrobotics/vision-service-examples).
```
if we do decide to capitalize should keep consistent

File: static/include/services/apis/generated/motion.md [hunk 14]
```diff
@@ -74,7 +74,7 @@ For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/
...
-- `destination` [(*referenceframe.PoseInFrame)](https://pkg.go.dev/go.viam.com/rdk/referenceframe#PoseInFrame): Describes where the `component_name` should end up. Can be any pose, from the perspective of any component whose location is configured as a [`frame`](../frame-system/). Note that the destination pose is relative to the distal end of the specified frame. This means that if the `destination` is the same as the `component_name` frame, for example an arm's frame, then a pose of `{X: 10, Y: 0, Z: 0}` will move that arm’s end effector by 10 mm in the local `X` direction.
+- `destination` [(*referenceframe.PoseInFrame)](https://pkg.go.dev/go.viam.com/rdk/referenceframe#PoseInFrame): Describes where the `component_name` should end up. Can be any pose, from the perspective of any component whose location is configured as a [`frame`](/services/frame-system/). Note that the destination pose is relative to the distal end of the specified frame. This means that if the `destination` is the same as the `component_name` frame, for example an arm's frame, then a pose of `{X: 10, Y: 0, Z: 0}` will move that arm’s end effector by 10 mm in the local `X` direction.
```
Comment (sguequierre) [inline static/include/services/apis/generated/motion.md:77]: this needs work, it only talks about translation when there's both translation and orientation https://pkg.go.dev/go.viam.com/rdk/spatialmath#Pose. can leave that for another time though

**Review by sguequierre — CHANGES_REQUESTED** [review]

File: docs/services/ml/deploy/tflite_cpu.md (hunk 11 above)
Comment (npentrel) [reply docs/services/ml/deploy/tflite_cpu.md:168]: I think I investigated this before and it's only detections and classifications but not 100% sure

File: docs/services/vision/_index.md (hunk 12 above)
[the remark this answers is not in the document]
Comment (npentrel) [reply docs/services/vision/_index.md:99]:
```suggestion
The vision service supports the following [vision service API](/appendix/apis/services/vision/) methods:
```

File: docs/services/vision/obstacles_distance.md (hunk 13 above)
Comment (npentrel) [reply docs/services/vision/obstacles_distance.md:140]: no - rule is not to capitalize, so the capitalization is wrong

File: static/include/services/apis/generated/motion.md (hunk 14 above)
Comment (npentrel) [reply static/include/services/apis/generated/motion.md:77]: yeah, I think that can happen when these code snippets get reviewed

**Comment by npentrel** [comment]
@sguequierre ready for another round - thanks!

File: docs/appendix/apis/services/base-rc.md [hunk 15]
```diff
@@ -0,0 +1,40 @@
...
+linkTitle: "Base Remote Control"
+weight: 70
+type: "docs"
+tags: ["base", "services", "rover", "input controller", "remote control"]
+description: "Give commands to get a list of inputs from the controller that are being monitored for that control mode."
+icon: true
+images: ["/services/icons/slam.svg"]
+---
+
+The Base Remote Control service API allows you to get a list of inputs from the controller that are being monitored for that control mode.
```
Comment (sguequierre) [inline docs/appendix/apis/services/base-rc.md:12]:
```suggestion
The base remote control service API allows you to get a list of inputs from the controller that are being monitored for that control mode.
```
if rule is not to capitalize should we not capitalize here?

File: docs/tutorials/projects/verification-system.md [hunk 16]
```diff
@@ -338,7 +338,7 @@ Now that you've got the verification aspect of your system working, you can use
 For example:
 
 - Write a program using one of the [Viam SDK](/sdks/) to poll the `facial-verification` module for its current state, and take action when a particular state is reached.
-  For example, you could use [`GetClassificationsFromCamera()`](/services/vision/#getclassificationsfromcamera) to capture when a transition into the `ALARM` state occurs, and then send you an Email with the captured image of the trespasser!
+  For example, you could use [`GetClassificationsFromCamera()`](/appendix/apis/services/vision/#getclassificationsfromcamera) to capture when a transition into the `ALARM` state occurs, and then send you an Email with the captured image of the trespasser!
```
Comment (sguequierre) [inline docs/tutorials/projects/verification-system.md:341]:
```suggestion
  For example, you could use [`GetClassificationsFromCamera()`](/appendix/apis/services/vision/#getclassificationsfromcamera) to capture when a transition into the `ALARM` state occurs, and then send you an email with the captured image of the trespasser!
```

File: static/include/services/apis/overrides/protos/motion.MoveOnMap.md [hunk 17]
```diff
@@ -17,7 +17,7 @@ Use the machine's position reported by the {{< glossary_tooltip term_id="slam" t
 {{< alert title="Requirements" color="info" >}}
 To use `MoveOnMap()`, your [SLAM service](/services/slam/) must implement `GetPointCloudMap()` and `GetPosition()`
 
-Make sure the [SLAM service](/services/slam/) you use alongside the this motion service supports the following methods in its {{< glossary_tooltip term_id="model" text="model's" >}} implementation of the [SLAM service API](/services/slam/#api):
+Make sure the [SLAM service](/services/slam/) you use alongside the this motion service supports the following methods in its {{< glossary_tooltip term_id="model" text="model's" >}} implementation of the [SLAM service API](/appendix/apis/services/slam/):
```
Comment (sguequierre) [inline static/include/services/apis/overrides/protos/motion.MoveOnMap.md:20]:
```suggestion
Make sure the [SLAM service](/services/slam/) you use alongside this motion service supports the following methods in its {{< glossary_tooltip term_id="model" text="model's" >}} implementation of the [SLAM service API](/appendix/apis/services/slam/):
```

**Review by sguequierre — CHANGES_REQUESTED** [review]

**Review by sguequierre — APPROVED** [review]
LGTM

**Pull request merged**